docs/machine-learning/resources/transforms.md

```diff
@@ -1,168 +1,102 @@
+| <xref:Microsoft.ML.TextCatalog.ProduceHashedNgrams*> | Transform text column into a vector of hashed ngram counts |
+| <xref:Microsoft.ML.TextCatalog.ProduceHashedWordBags*> | Transform text column into a bag of hashed ngram counts |
+| <xref:Microsoft.ML.TextCatalog.RemoveDefaultStopWords*>  | Remove default stop words for the specified language from input columns |
+| <xref:Microsoft.ML.TextCatalog.RemoveStopWords*> | Removes specified stop words from input columns |
+| <xref:Microsoft.ML.TextCatalog.LatentDirichletAllocation*> | Transform a document (represented as a vector of floats) into a vector of floats over a set of topics |
+| <xref:Microsoft.ML.TextCatalog.ApplyWordEmbedding*> | Convert vectors of text tokens into sentence vectors using a pre-trained model |
 
-## Onnx
+## Image transformations
 
 | Transform | Definition |
 | --- | --- |
-| <xref:Microsoft.ML.Transforms.OnnxTransform> | Scores pre-trained ONNX models which use the ONNX standard v1.2 |
+| <xref:Microsoft.ML.ImageEstimatorsCatalog.ConvertToGrayscale*> | Convert an image to grayscale |
+| <xref:Microsoft.ML.ImageEstimatorsCatalog.ConvertToImage*> | Convert a vector of pixels to ImageType |
+| <xref:Microsoft.ML.ImageEstimatorsCatalog.ExtractPixels*> | Convert pixels from input image into a vector of numbers |
+| <xref:Microsoft.ML.ImageEstimatorsCatalog.LoadImages*> | Load images from a folder into memory |
+| <xref:Microsoft.ML.ImageEstimatorsCatalog.ResizeImages*> | Resize images |
 
-## Preprocessing
-| Transform | Definition |
-| --- | --- |
-| <xref:Microsoft.ML.Transforms.BootstrapSamplingTransformer> | Approximates bootstrap sampling using Poisson sampling. |
-| <xref:Microsoft.ML.Transforms.Projections.RandomFourierFeaturizingTransformer> | Produces random Fourier feature. |
-| <xref:Microsoft.ML.Transforms.Text.TokenizingByCharactersTransformer> | Character-oriented tokenizer where text is considered a sequence of characters. |
-| <xref:Microsoft.ML.Transforms.Projections.VectorWhiteningTransformer> | Simplifies optimization to assist with identifying weights. |
-
-## Row Filters
+## Categorical data transformations
 
 | Transform | Definition |
 | --- | --- |
-| <xref:Microsoft.ML.Transforms.RowShufflingTransformer> | Shuffles a randomized cursor attempt to perform using a pool of a given number of rows.  |
-| <xref:Microsoft.ML.Transforms.SkipFilter> | Allows limiting input to a subset of rows by skipping a number of rows. |
-| <xref:Microsoft.ML.Transforms.SkipTakeFilter> | Allows limiting input to a subset of rows at an optional offset. Can be used to implement data paging. When created with SkipTakeFilter.SkipArguments behaves as `SkipFilter`.
-| <xref:Microsoft.ML.Transforms.TakeFilter> | Allows limiting input to a subset of rows by taking N first rows. |
-
-## Schema
+| <xref:Microsoft.ML.CategoricalCatalog.OneHotEncoding*> | Convert one or more text columns into one-hot encoded vectors |
```

Comment:
do you think one-hot should link to something that explains it?

Reply:
Added link